packages/neo-one-server-plugin-wallet/src/__e2e__/bootstrap.test.js

```diff
@@ -0,0 +1,78 @@
+/* @flow */
+import { addressToScriptHash } from '@neo-one/client';
+
+describe('bootstrap', () => {
+  test.only('bootstrap on active network - default', async () => {
+    const numWallets = 10;
+
+    await one.execute('create network boottest');
+    await new Promise(resolve => setTimeout(() => resolve(), 10000));
+
+    await one.execute('bootstrap');
+    await new Promise(resolve => setTimeout(() => resolve(), 10000));
+
+    const output = await one.execute('get wallet --json');
```

Comment:
Same here

Reply:
And everywhere else in this file. Remember, this is running as just a normal command, not from the interactive cli, so no network is activated.